app/helpers/arclight_helper.rb

```diff
@@ -113,6 +113,19 @@ def display_size(size)
     size.to_s
   end
 
+  # determine which icon to show in search results header
+  # these icon names will need to be updated when the icons are determined
+  def document_header_icon(document)
+    level = document['level_ssm'] || nil
```

Comment:
Can we just use the accessor already available on `document`?

```ruby
document.level
```

Reply:
updated to use document.level